RELEASE_NOTES.md

```diff
@@ -24,5 +26,6 @@ Finally, when using the `SetOrder*` functions in simapp, e.g. `SetOrderBeginBloc
 - Speedup improvements (e.g. speedup iterator creation after delete heavy workloads, lower allocations for `Coins.String()`, reduce RAM/CPU usage inside store/cachekv's `Store.Write`) are included in this release.
 - Upgrade Rosetta to v0.7.0 .
 - Support in-place migration ordering.
+- Copied and updated `server.GenerateCoinKey` and `server.GenerateServerCoinKey` functions to the `testutil` package. These functions in `server` package are marked deprecated and will be removed in the next release. In the `testutil.GenerateServerCoinKey` version we  added support for custom mnemonics in in-process testing network.
```

Comment:
Can we mark in bold that this is an API breaking change?

Reply:
This is not API breaking. The "new" function is updated, but the old has only tiny cleanups, without breaking anything. The api breaking part is above (TestMnemonic).

Reply:
Ah you're right 👍